dist/index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Restaurant page</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1"></head>
+  <body>
```

Comment:
Well done developer.... 👍 
According to the instruction on `Odin` page, we should have `<div id="content"></div>` inside the `body` tag. 
![image](https://user-images.githubusercontent.com/31897434/88849163-66288600-d1e1-11ea-8414-32df71189e88.png)

Reply:
Hello Oluwaseun Bello :raised_hands: 

Thanks for all your feedback, I accomplish the changes to ask for, and go to re-submit for review :+1: :roller_coaster: